src/main/java/com/labsynch/labseer/service/StandardizationServiceImpl.java

```diff
@@ -401,7 +401,7 @@ private String getParentAlias(Parent parent) {
 
 	@Override
 	@Transactional
-	public int dupeCheckStandardizationStructures() throws CmpdRegMolFormatException {
```

Comment:
This compiles properly, right? So we don't need to change the abstract interface at StandardizationService?

I recall having to tweak both places, but maybe it is OK with an implementation that just doesn't throw some of the exception types.